### ISSUE TITLE
Comet automatically adding Cysteine Static Modification

### ISSUE THREAD

**Issue by mobiusklein** [crux-toolkit/crux-toolkit#485]
After upgrading to Crux `3.2-58ce7667` from `2.1.17010`, I noticed a new message appearing in the search log:
```
DEBUG: Added new modification [42e1d5a0]: aa[C] dM[57.021464] static[1] symbol[ ]
```
Since this message exactly matches the `carp` message in https://github.com/crux-toolkit/crux-toolkit/blob/d330b2f41f3f3961c1c31d966148ce5288bfe602/src/model/Modification.cpp#L808, I think this is probably `crux`-related instead of `comet`-specific. 

Both `tide` and `kojack` have a note in https://github.com/crux-toolkit/crux-toolkit/blob/5444d7fda9368aac21a3618b19967af067117ec9/src/util/Params.cpp that says if a cysteine modification isn't specified a fixed modification of 57.021 is added automatically like this, but say that if you specify a cysteine modification the automatic modification is not added.

I tried specifying a `variable_mod` that is variable or binary, or a static modification on C to see if it woud remove that `DEBUG` message noted above, but the message continued to appear.

**Comment by wsnoble** [comment]
I agree that this seems to be problematic. It is probably related to the
Comet upgrade and its integration into Crux. Can you send an example that
displays the problem (ideally, input files and log file)? That will help us
debug.
Bill

**Comment by mobiusklein** [comment]
The parameter file is simple enough:
[comet_params_modC.txt](https://github.com/crux-toolkit/crux-toolkit/files/5804425/comet_params_modC.txt)

I'm just searching the human reference proteome against a proprietary dataset. I doubt those are what trigger the static modification though?

**Comment by rita-gwen** [comment]
Joshua,
apparently, crux also has an additional set of parameters to specify static mods. They are just single letters, one  per amino acid. And -C parameter has default value of 57.021464, logically enough. (The rest of them default to 0) 
So when you specify a static mod of 0 on cysteine using the comet parameter it is still overwritten by that default. 
It looks like these parameters are not used by any of crux applications so we are in process of removing them, but for now you can just add 
C=0 
line at the beginning of your parameter file as a workaround and it should take care of that message.
I'll let you know when the fixed version will be available to download.

**Comment by mobiusklein** [comment]
Thank you. I saw something like that when I did my first dive into the codebase, but I wasn't sufficiently familiar with the option parser and I thought it was somehow related to `add_C_cysteine`.
